packages/wallet/package.json

```diff
@@ -103,13 +107,9 @@
 		"@types/react-redux": "^7.0.0",
 		"@types/react-sidebar": "3.0.0",
 		"@types/reactstrap": "^8.0.0",
-		"@types/web3": "^1.0.2",
-		"@babel/runtime": "7.6.2",
-		"@statechannels/devtools": "0.1.21",
-		"@types/jest": "24.0.18",
-
 		"@types/redux-storage": "4.1.0",
 		"@types/storybook__react": "4.0.0",
+		"@types/web3": "^1.0.2",
```

Comment:
Do you know where we're still using this? Is it possible to completely remove in favour of ethers?

Reply:
I think this can go